### ISSUE TITLE
Read Only (and a strange error in the console)

### ISSUE THREAD

**Issue by nicoesiea** [ruilisi/fortune-sheet#106]
**Describe the bug**
I'm looking for how to display an excel file in ready only mode - without any edition

**To Reproduce**
I open the file using LuckExcel, directly from an URL with **LuckyExcel.transformExcelToLuckyByUrl**
Then I load the result with the following:
````
let options = {
          container: 'react-excel-id', //react-excel-id
          showinfobar: false,
          showsheetbar: true,
          data: exportJson.sheets,
          enableAddRow: false,
          row: 26,
          column: 26,
          allowUpdate: false,
          enableAddBackTop: false,
          lang: this.translateService.getDefaultLang()
        };
        ReactDOM.render(
          React.createElement(ExcelReactComponent, options),
          document.getElementById('react-excel-id')
        );
[...]
// Then I load the component with:
<Workbook data={props.data} config={props} />
````

With this way, I can see the following error on my console:
![image](https://user-images.githubusercontent.com/4542315/204554350-1fe26c91-3a7b-48d9-a6bb-5f8e9ed95df9.png)

**Expected behavior**
Is there an issue with my code? It is possible to block the edition?

**Screenshots**
![image](https://user-images.githubusercontent.com/4542315/204554811-7b39aa13-cff1-415e-a71c-b0752c8ea7e1.png)

**Additional context**
I'm using Angular 13 with the react plugin loaded with a wrapper (just for you to know)

**Comment by nicoesiea** [comment]
I found the answer for edit mode, I have to add allowEdit as Workbook parameter:
````<Workbook data={props.data} lang={props.lang} allowEdit={props.allowEdit} />````
I add the property in the **options** object and it works as expected

But I still have the console error:
![image](https://user-images.githubusercontent.com/4542315/204560156-83778ab3-4258-446c-960a-527f0187cafe.png)

**Comment by nicoesiea** [comment]
The error is no more visible if I hide the toolbar: **showToolbar: false,**
I do believe it is a bug related to each item on this bar

**Comment by shuanhukexuejia** [comment]
The warning has been fixed in #102 , will be included in next version
